src/App/Fossa/ProjectInference.hs

```diff
@@ -78,7 +79,7 @@ inferSVN dir = do
               . dropPrefix rootRelativeToUrl
               . dropPrefix "^" $ relUrl
 
-        pure $ InferredProject root revision (if T.null trimmedRelative then "trunk" else trimmedRelative)
+        pure $ InferredProject root revision (Just $ if T.null trimmedRelative then "trunk" else trimmedRelative)
```

Comment:
We hard-code `trunk` here for SVN.  Should we also remove this logic?

Reply:
Yes 👍